docker/reqmgr2ms/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cmssw/cmsweb:20210621
+FROM cmssw/cmsweb:20210621 AS reqmgr2ms
```

Comment:
I"m not sure why did you make this change. Since you are not changing anything in reqmgr2ms image you don't need it

Reply:
ok, removing it than